src/components/auth/LoginPage.scss

```diff
@@ -0,0 +1,24 @@
+.LoginForm {
+    margin-top: 20px;
+    width: 80%;
+    max-width: 540px;
+    padding: 0 5%;
```

Comment:
I wouldn't use % in padding. I guess the effect of the padding is only visible on mobile anyway just choose some fixed you find nice for mobile and stick to it.